### ISSUE TITLE
Adding a new style doesn't always work the 1st time around

### ISSUE THREAD

**Issue by markpenny** [sillsdev/ptx2pdf#513]
I'm not sure where I went wrong, but the 1st time I tried adding a Style, I ended up with an error message like this:
![image](https://user-images.githubusercontent.com/12962708/132779383-724add3d-4e06-46ba-bb17-805e20c06163.png)

On investigating further, the issue was discovered in the ptxprint.sty file which showed an incomplete definition:
```
\Marker zim
\LineSpacing 1
\FontSize 10.68
\Italic 
\LeftMargin 0.4
\RightMargin 0.4

``` 
After deleting it and re-trying to add the marker, it was successful:
```
\Marker zim
\Name zim - zim - Paragraph - Margin Intro
\Description For INT book special text
\TextType VerseText
\StyleType Paragraph
\LeftMargin 0.3
\RightMargin 0.3
\Italic 
\FontSize 10.68
\BaseLine 11 pt

```
So I'm not sure why it was bad/different the first time I tried it. Maybe because the name didn't conform to what it was expecting. Perhaps we need a "template" +/- validation and/or tooltips for the name field when adding a new marker.

At the very least we should add place-holder text in the "form", and Tooltips to explain what is expected in each.

**Comment by mhosken** [comment]
I'm not sure how you managed to get nothing into those fields, but I've just committed code to ensure you can't not specify required fields and labelled them in the UI. It's probably a bit too unsubtle. Help in that area is appreciated.

**Comment by markpenny** [comment]
Yes, I'm happy to talk about it and bounce ideas around the next time we chat. Surely 'Name' also needs something in there too... otherwise how does it categorize it?

**Comment by mhosken** [comment]
Name is optional, although we should fill it in with *something* if it is blank. Uncategorised markers go in Other at the bottom of the list.

**Comment by markpenny** [comment]
Yes, that is working much better now, and forces users to insert data into the required fields. I also added placeholder text to indicate what kinds of info they need to enter.

However, I noticed that if I EDIT a marker (after creating it), the TreeView description doesn't get updated until I close and re-open PTXprint. Is that something that is easy to fix? I hope so:
![image](https://user-images.githubusercontent.com/12962708/133005242-8606261c-b317-430e-858c-7d011dea5ae6.png)
